.env

```
@@ -1 +1,3 @@
-VITE_BAT_CAVE_WALLPAPER = https://images3.alphacoders.com/160/thumb-1920-160927.jpg
\ No newline at end of file
+VITE_BAT_CAVE_WALLPAPER = https://images3.alphacoders.com/160/thumb-1920-160927.jpg
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)